src/pages/ReimbursementAccount/BusinessInfo/substeps/PhoneNumberBusiness.tsx

```diff
@@ -44,6 +44,8 @@ function PhoneNumberBusiness({reimbursementAccount, onNext, isEditing}: PhoneNum
     const handleSubmit = useReimbursementAccountStepFormSubmit({
         fieldIds: STEP_FIELDS,
         onNext,
+        // We want to remove sanitize user input i.e. remove leading and trailing whitespaces
```

Comment:
Suggestion:

> // During draft saving, the phone number is sanitized (i.e. leading and trailing whitespace is removed)

- Link between draft saving and sanitization
- Small language mistake fixed (whtespace**s**)
- Stressing that we want this for the phone numbers specifically